tools/scenario-geojson2scn.py

```diff
@@ -0,0 +1,240 @@
+import argparse
+import json
+import os
+import unicodedata
+from datetime import timedelta
+from typing import List
+
+uas_id = 0
+area_id = 0
+
+# TODO comment code
+
+defaults = {
+    "zoom": 16,
+    "duration": 30,
+    "plugins": ["dronetraf", "usepelogger"],
+    "setup": [
+        "dronetraf area traffic_area",
+        "asas on",
+        "reso mvp",
+        "rmethh spd",
+        "usepeconflog on",
```

Comment:
Changing this to `usepelogger on` will automatically turn on all data loggers.
```suggestion
        "usepelogger on",
```